beatrun/gamemodes/beatrun/gamemode/cl/ButtonHints.lua

```diff
@@ -0,0 +1,185 @@
+-- This file has lots of strings that needs to be localized.
```

Comment:
I think you can remove this line

Reply:
Oh yeah I forgot about that, this *does* support localization now.